partials/tabs/languages.html

```diff
@@ -219,4 +220,4 @@
     $('.tooltipped').tooltip({delay: 50});
 
     })
-</script>
+</script>
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp8v872ato/partials/tabs/languages.html
+++ b/tmp/tmp8v872ato/partials/tabs/languages.html
@@ -220,4 +220,4 @@
     $('.tooltipped').tooltip({delay: 50});
 
     })
-</script>+</script>
```